design/definitions.go

```diff
@@ -652,9 +664,12 @@ func (r *ResourceDefinition) DSL() func() {
 }
 
 // Finalize is run post DSL execution. It merges response definitions, creates implicit action
-// parameters, initializes querystring parameters and sets path parameters as non zero attributes.
+// parameters, initializes querystring parameters, sets path parameters as non zero attributes
+// and sets the fallbacks for security methods.
 func (r *ResourceDefinition) Finalize() {
 	r.IterateActions(func(a *ActionDefinition) error {
+		a.Finalize()
```

Comment:
I understand.. 3 lines later we do the same with Responses so I figured the pattern was: one or the other. I don't mind using the DefinitionSet instead. Just tell me what you prefer going forward.